tests/workloads/spin-runtime-config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+data:
+  runtime-config.toml: "# This defines a new store named user_data\n[key_value_store.foo]\ntype
```

Comment:
nit: you can do 

```
config.yml: |-
  # This defines a new store named user_data
  [key_value_store.foo]
  type = "spin"  
  path = "user_data.db"

Reply:
Question: is this config checked anywhere in the tests to ensure it is used? I am not seeing it but might have missed it.

Reply:
I believe `let store = Store::open("foo")?;` is using the `foo` store defined in the `runtime-config.toml`. The `images/spin-keyvalue/Dockerfile` does not copy runtime config file to the image, so it must have been reading the config from the ConfigMap mounted as a file.

Reply:
Yeah this file is generated from `kubectl create configmap spin-runtime-config --from-file=runtime-config.toml --output=yaml`